database/src/main/java/com/firebase/ui/database/OnChangedListener.java

```diff
@@ -0,0 +1,41 @@
+package com.firebase.ui.database;
+
+import com.google.firebase.database.ChildEventListener;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+
+public interface OnChangedListener {
```

Comment:
But then isn't this essentially doing the same: making a part of the internal workings of FirebaseArray public?